src/content/blog/2024/office_hours.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Introducing nf-core office hours'
+subtitle: "A new initative from nf-core! Here's what you can expect."
+headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
+headerImageAlt: 'Person using macbook pro and holding cappuccino'
+announcement:
+    text: 'Introducing nf-core office hours'
+    start: 2024-01-12T16:14:00+01:00
+    end: 2024-01-13T16:14:00+01:00
+pubDate: 2024-01-12T16:14:00+01:00
+authors:
+    - 'christopher-hakkaart'
+label:
+    - 'community'
+---
+
+# nf-core office hours
+
+Slack serves as an outstanding platform for asynchronous collaboration.
+Nonetheless, there are moments when engaging in live discussions about your code proves beneficial.
+We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers seeking an opportunity to connect and chat.
+
+# What you can expect
+
+Office hours will be most useful for “advanced beginners” - people who have grasped the fundamentals of writing nextflow pipelines, and are now trying to make the leap to designing, coding, and contributing nf-core components and pipelines.
+However, regardless of what stage you are at, if you think that spending some time talking about developing with nf-core sparks your interest, then the office hours might be just what you're looking for!
+
+In these sessions, some of the nf-core maintainer team and other seasoned community members will be available to engage in discussions about your code.
```

Comment:
Did we check with the maintainers team they are OK contributing to this?

Reply:
I'll circulate in the channel and join the next meeting for feedback.